src/UserCollectableAttribute.js

```diff
@@ -151,6 +151,51 @@ class UserCollectableAttribute {
     return _.clone(definition);
   }
 
+  static fromFlattenValue(identifier, values) {
+    const meta = UserCollectableAttribute.getUCAProps(identifier);
+    const fixedValues = _.map(values, (item) => {
+      const fixedValue = _.cloneDeep(item);
+      const nameComponents = _.split(item.name, '>');
+      const name = nameComponents[0];
+      const sufix = nameComponents[1];
```

Comment:
the `getFlattenValue` method allows the serialization of objects with any level of nesting. The fromFlatternValue will not be able to build the value if the object is too complex, right?
But I think it is ok as long as we are not expecting to have UCAs with very sophisticated properties.

Reply:
No, I belive the implementation works for both